src/style.css

```diff
@@ -1,6 +1,66 @@
+/* General */
+
+body {
```

Comment:
- [ ] I couldn't see if the project show the required style because there's no package.json file. I invite you to add it to see if the project has the elements. 🐶